XilinxProcessorIPLib/drivers/llfifo/examples/xllfifo_polling_example.c

```diff
@@ -272,7 +275,7 @@ int TxSend(XLlFifo *InstancePtr, u32  *SourceAddr)
 
 	/* Filling the buffer with data */
 	for (i=0;i<MAX_DATA_BUFFER_SIZE;i++)
-		*(SourceAddr + i) = 0;
+		*(SourceAddr + i) = i;
```

Comment:
Fix done (i.e RDFO read should be before RLR) in interrupt example is required in the poll example as well.